org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/instance/InstanceValidator.java

```diff
@@ -1015,6 +1023,30 @@ public void validate(Object appContext, List<ValidationMessage> errors, String p
     codingObserver.finish(errors, stack);
     errors.removeAll(messagesToRemove);
     timeTracker.overall(t);
```

Comment:
Should AI run after time tracking for overall validation, or should AI be included in overall time?